client_authentication_test.go

```diff
@@ -512,6 +537,28 @@ func TestAuthenticateClient(t *testing.T) {
 	}
 }
 
+// func TestCheckClientSecret(t *testing.T) {
```

Comment:
Do you want to enable this test or remove it? :)

Reply:
Tried to clean commit history and somewhat ended up with this there :)

Removed it now, but it relates to my last question.